crypto/bio/b_sock.c

```diff
@@ -387,6 +387,8 @@ int BIO_socket_wait(int fd, int for_read, time_t max_time)
     struct timeval tv;
     time_t now;
 
+    if (fd < 0 || fd >= FD_SETSIZE)
```

Comment:
Okay, but I'd like to see the `fd >= FD_SETSIZE` also in bio_wait, and defer to the default handling
in that case.

Reply:
Good point - done.

On this occasion I've also renamed the `millisecondss` param of `BiO_wait()` to `fallback_milliseconds` for clarity.